test/Processors/HookExecutionProcessorTests.cs

```diff
@@ -64,38 +64,6 @@ public void Setup()
             };
         }
 
-        //[BeforeScenario("Foo")]
```

Comment:
This wasn't commented code :) - this bit of code block was to indicate what is the logic being tested. It was to ease  the data setup. Perhaps you can add a line describing such, but i prefer we leave this in place.